dag-sync.sh

```diff
@@ -20,7 +20,13 @@ git fetch origin
 new=$(git rev-list --reverse --topo-order HEAD..origin/main | head -1)
 # If there is no new commit hash to move to, nothing has changed, quit early
 [ -z "$new" ] && exit
-# Move ahead to this new commit
+
+# Verify if have /dags/ in the last commit
+have_dag=$(git log -p -1 "$new"  --pretty=format: --name-only | grep "/dags/")
+# If there is no files under /dags/ folder, no need to notify, quit early
+[ -z "$have_dag" ] && exit
```

Comment:
One more change I just realized (sorry!), this should go _after_ the `git reset --hard "$new"` line below. If we don't move forward on each commit, `$new` will always be the next commit ahead and we will run into the same loop of logic for skipping it if it happens to not include DAGs. We only want to exit prior to sending the Slack notification if there aren't any DAGs.